ArduCopter/land_detector.cpp

```diff
@@ -49,12 +49,9 @@ void Copter::update_land_detector()
 #if FRAME_CONFIG == HELI_FRAME
         // if rotor speed and collective pitch are high then clear landing flag
         if (motors->get_takeoff_collective() && motors->get_spool_state() == AP_Motors::SpoolState::THROTTLE_UNLIMITED) {
-#else
```

Comment:
We need to do the same thing for Heli.

Reply:
I would have to add an additional condition here that would add non manual throttle for this statement because I don't clear the land complete flag in acro or stabilize.  It is all done in this method.